airtime_mvc/application/configs/conf.php

```diff
@@ -54,6 +41,19 @@ public static function loadConfig() {
             $CC_CONFIG['dev_env'] = 'production';
         }
         
+        // Parse separate conf file for cloud storage values
+        $cloudStorageConfig = "/etc/airtime-saas/".$CC_CONFIG['dev_env']."/cloud_storage.conf";
+        $cloudStorageValues = parse_ini_file($cloudStorageConfig, true);
+        
+        $CC_CONFIG["supportedStorageBackends"] = array('amazon_S3');
+        foreach ($CC_CONFIG["supportedStorageBackends"] as $backend) {
+            $CC_CONFIG[$backend] = $cloudStorageValues[$backend];
+        }
+        
+        // Tells us where file uploads will be uploaded to.
+        // It will either be set to a cloud storage backend or local file storage.
+        $CC_CONFIG["current_backend"] = $cloudStorageValues["current_backend"]["storage_backend"];
+        
```

Comment:
Can you make this code do one more thing:
- Default to the "production" cloud_storage.conf if a per-environment conf doesn't exist?

If this was in regular Airtime, I'd also say that we should make the code still work and fallback to file storage if no cloud_storage.conf was found at all. However, I think that since this code is saas-only and we ONLY want S3 uploads now, I'd rather have the code fail out (like it does now) if a cloud_storage.conf wasn't found. (If that file doesn't exist at all, then it's a server configuration issue and we'd want to know about that right away, rather than having files silently be stored on disk again.)